### PR TITLE
On load, don't show error message

### PR DESCRIPTION
**Pull request by jbydeley-d2l**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import {createStore} from 'store/index'
 import * as types from 'store/mutation-types'
 import {i18n} from 'i18n'
 
+Vue.config.performance = process.env.NODE_ENV !== "production"
```
Comment (jwong-d2l) [inline src/main.js]: what does this guy do?

File: test/components/UserSearch_test.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+			UserSearch = new Constructor({
+				store,
+				i18n
+			}).$mount();
+			console.log("Mounted");
+		});
+
+		it("should exist by default", done => {
+			Vue.nextTick(() => {
+				console.log(UserSearch.$el);
```
Comment (jwong-d2l) [inline test/components/UserSearch_test.js]: are console.logs okay for tests?

File: src/main.js (hunk 1 above)
Comment (jbydeley-d2l) [reply src/main.js]: In development it allows the dev tools to collect performance information about the components. In production, it does nothing.

File: test/components/UserSearch_test.js (hunk 2 above)
Comment (jbydeley-d2l) [reply test/components/UserSearch_test.js]: Good point. They are fine but this one doesn't serve a purpose so I'll remove it.

Comment (jwong-d2l) [reply test/components/UserSearch_test.js]: follow up, should we lint our test files?

**Pull request merged**